src/services/YotiService.ts

```diff
@@ -16,10 +16,24 @@ export class YotiService {
 
   readonly PEM_KEY: string;
 
+  private readonly CLIENT_SESSION_TOKEN_TTL = process.env.CLIENT_SESSION_TOKEN_TTL;
+
+  private readonly RESOURCES_TTL = process.env.RESOURCES_TTL;
+
   constructor(logger: Logger, CLIENT_SDK_ID: string, PEM_KEY: string) {
-  	this.logger = logger;
-  	this.CLIENT_SDK_ID = CLIENT_SDK_ID;
-  	this.PEM_KEY = PEM_KEY;
+	  if (!this.CLIENT_SESSION_TOKEN_TTL
```

Comment:
if these could be move to EnvironmentVariables.ts pls

Reply:
since the EnvironmentVariables.ts is specific to govNotify related env variables, initializing this class and using it may fail incase it doesnt find the variables related to GovNotify not set in the environment.